src/main/webapp/pages/announcements/details.html

```diff
@@ -27,11 +27,27 @@
                     <h2 class="panel-title" th:text="#{announcement.control.details.edit}"></h2>
                 </div>
                 <div class="panel-body">
-                    <textarea class="form-control input-full announcement-input" type="text" id="message" name="message"
+                    <div class="form-group">
+                        <label for="title" th:text="#{announcement.control.title}">__Title__</label>
+                        <input id="title" name="title" type="text"
+                               class="form-control input-full"
+                               required="required"
+                               th:value="${announcement.getTitle()}"
+                               th:placeholder="#{announcement.create.form.placeholder}" />
+                    </div>
+                    <div class="form-group">
+                        <label for="message" th:text="#{announcement.control.message}">__Message__</label>
+                        <textarea class="form-control input-full announcement-input" type="text" id="message" name="message"
```

Comment:
I am trying to keep this page look and feel consistent. I have not been able to find the input-full class in the css.

Reply:
Well this brings me to my next point.  We need to get rid of this page :)  Let's chat about it.

Reply:
Okies!